Gopkg.toml

```diff
@@ -44,6 +44,10 @@
   name = "github.com/docker/docker-credential-helpers"
   version = "v0.6.3"
 
+[[override]]
+  name = "github.com/docker/cli"
+  revision = "23446275646041f9b598d64c51be24d5d0e49376"
```

Comment:
Is there maybe a release we can pin?

Reply:
For what I see the closest tag is `v19.03.0-beta1`. I can try with it.

Reply:
I've taken a dependency on this branch in the WIP for cleaning up dependencies in `duffle` and `cnab-go`, and got `v19.03.04`:

Not sure if something else is needed in this PR that would not be covered by that version - I'm happy with the dependencies in those two PRs, so if you think trying to pin a release would make things more difficult, I'm happy to go with the current state.

https://github.com/deislabs/cnab-go/blob/cf65be2299bfd86475cd1a68869ae94611f47416/Gopkg.lock#L242

Reply:
Quick update about this one (I haven't forgotten it :-) )
I'm working on docker app based on this branch for several days, and it works pretty well (at least on dependency side).
So I suggest to move forward with this version now.

And a little after, I'll come back with a whole update, not only about docker/cli but also with right and coordinated versions of docker/distribution and docker/docker (where we cannot have tags at he moment).

Reply:
Yeah, I've also based work in both `duffle` and `cnab-go` on this branch, so I think we can go ahead and merge this.

Let's keep in mind an update for CLI in the future.

Thanks a lot for the work here!